subsys/partition_manager/pm.yml.trustzone

```diff
@@ -4,6 +4,7 @@ sram_secure:
   region: sram_primary
 
 sram_nonsecure:
+  span: [sram_primary] /* Here, sram_primary refers to the (NS) app's RAM. */
   region: sram_primary
   span:
```

Comment:
if this commit d15cf99ac0c40ae8cc78521ffb445d718d616238 is a revert of 07d8aed4f1508378a6d41c3d0fa6ace1ff4daf2b

then why are those lines still present:
https://github.com/nrfconnect/sdk-nrf/blob/d15cf99ac0c40ae8cc78521ffb445d718d616238/subsys/partition_manager/pm.yml.trustzone#L9-L13

Reply:
Cleaned up